modules/libcom/test/iocshTestNullArgument.cmd

```diff
@@ -0,0 +1,4 @@
+on error break
+position before_error
+epicsThreadSleep \0
```

Comment:
This line isn't doing what you think.  This is passing `"\\0"`.  Pass nothing to pass an NULL.  eg. just `epicsThreadSleep` with no number.  That said, this does doesn't cover the change to `iocsh.cpp` as argument type of epicsThreadSleep isn't iocshArgPersistentString.  (as mentioned on the LP bug, there are no internal users)

Reply:
Thanks. This clarifies my previous message.